Src/Common/CorrelationIdLookupHelper.cs

```diff
@@ -119,22 +122,36 @@ public bool TryGetXComponentCorrelationId(string instrumentationKey, out string
                             {
                                 this.GenerateCorrelationIdAndAddToDictionary(instrumentationKey, appId.Result);
                             }
+#if !NETCORE
                             catch (Exception ex)
                             {
                                 CrossComponentCorrelationEventSource.Log.FetchAppIdFailed(this.GetExceptionDetailString(ex));
                             }
+#else
+                            catch (Exception)
```

Comment:
How would the user go about debugging if say the endpoint to get app ID is down. Shouldn't we log this somewhere even for .net core?